test/functional/update/slow-server/test.bats

```diff
@@ -8,6 +8,11 @@ port=""
 targetfile="6c27df6efcd6fc401ff1bc67c970b83eef115f6473db4fb9d57e5de317eba96e"
 
 setup() {
+  # Skip if not in travis env
```

Comment:
Longer comment explaining why we only run the test in Travis (too slow to run for local development) with an explanation of how to run it locally (`TRAVIS=foo make check`) please.